src/images/star-active.svg

```diff
@@ -1 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16.93 16.49"><defs><style>.cls-1{fill:#f16139;}</style></defs><title>Artboard 30</title><g id="background"><polygon class="cls-1" points="8.28 2.85 9.91 6.13 13.53 6.66 10.91 9.22 11.53 12.83 8.28 11.13 5.04 12.83 5.66 9.22 3.03 6.66 6.66 6.13 8.28 2.85"/></g></svg>
```

Comment:
Removing the SVGs was a good call. PNGs are a lot less of a headache with React 😅